modyn/supervisor/internal/grpc_handler.py

```diff
@@ -101,7 +75,6 @@ def init_cluster_connection(self) -> None:
         self.init_evaluator()
 
     def init_storage(self) -> None:
-        assert self.config is not None
```

Comment:
I think these assertions played a role in some unit tests, but if it still works and they don't throw i guess let's remoev them

Reply:
the type if `self.config` is already non-optional. And it is directly initialized in constructor with a non-optional modyn_config. So I think we are safe